deep_qa/layers/attention/gated_attention.py

```diff
@@ -124,3 +123,8 @@ def call(self, inputs, mask=None):
             raise ConfigurationError("Invalid gating function "
                                      "{}, expected one of {}".format(self.gating_function,
                                                                      GATING_FUNCTIONS))
+
+    def get_config(self):
+        config = {'gating_function': self.gating_function}
+        base_config = super(GatedAttention, self).get_config()
+        return dict(list(base_config.items()) + list(config.items()))
```

Comment:
I guess I should note that keras uses `nb_` instead of `num_` across their codebase.  That doesn't mean we should make the same poor choice...